main.tf

```diff
@@ -1 +1,162 @@
-# main.tf placeholder
+#
+# Deny root account
+#
+
+data "aws_iam_policy_document" "deny_root_account" {
+  statement {
+    actions   = ["*"]
+    resources = ["*"]
+    condition {
+      test     = "StringLike"
+      variable = "aws:PrincipalArn"
+      values   = ["arn:aws:iam::*:root"]
+    }
+  }
+}
```

Comment:
Is the thought you'd use this on the child accounts?

Reply:
yeah, there should be no reason to use the root user in any of the child accounts.